tensorflow/python/compiler/tensorrt/tensorrt.bzl

```diff
@@ -0,0 +1,27 @@
+load("//tensorflow:tensorflow.bzl", "cuda_py_test")
+
+def tensorrt_py_test(test_names, tags):
+    # cuda_py_test enable XLA tests by default. TensorRT can't combine XLA with
+    # TensorRT currently and should set xla_enable_strict_auto_jit to False to
+    # disable XLA tests.
```

Comment:
The previous comment should still applies here:
 Please add a docstring for the routine.
 please also add "# buildifier: disable=unnamed-macro" right before the function "def tensorrt_py_test ..." to silent this warning:
The macro "tensorrt_py_test" should have a keyword argument called "name".

It is considered a macro because it calls a rule or another macro "cuda_py_test" on line 15.

Reply:
DONE